test/hotspot/jtreg/vmTestbase/nsk/monitoring/ThreadMXBean/resetPeakThreadCount/reset001.java

```diff
@@ -21,6 +21,48 @@
  * questions.
  */
 
+/*
+ * @test
+ *
+ * @summary converted from VM Testbase nsk/monitoring/ThreadMXBean/resetPeakThreadCount/reset001..reset005
+ * VM Testbase keywords: [quick, monitoring]
+ * VM Testbase readme:
+ * DESCRIPTION
+ *     The test checks that
+ *         ThreadMXBean.resetPeakThreadCount()
+ *     returns correct result.
+ *     The test starts a couple of user threads and waits until they finish. After
+ *     that, resetPeakThreadCount() is invoked to reset the peak. Then
+ *     getPeakThreadCount() and getThreadCount() must return the same values. The
+ *     preposition is that no threads are appered/disappeared between
+ *     "getPeakThreadCount()" and getThreadCount()" calls.
+ *     The test implements defferent ways to access to the metrics.
+ *
+ * @library /vmTestbase
+ *          /test/lib
+ *
+ * @comment Direct access to the metrics.
+ * @run main/othervm nsk.monitoring.ThreadMXBean.resetPeakThreadCount.reset001
```

Comment:
I think it would be better to have several @test id instead of multi @run for such cases. It allows for reporting the individual results for each run and executing them parallelly.

Reply:
Good point. fixed.